tests/components/demo/test_media_player.py

```diff
@@ -203,7 +204,7 @@ async def test_seek(hass, mock_media_seek):
     await hass.async_block_till_done()
     ent_id = "media_player.living_room"
     state = hass.states.get(ent_id)
-    assert state.attributes["supported_features"] & mp.SUPPORT_SEEK
+    assert state.attributes[ATTR_SUPPORTED_FEATURES] & mp.SUPPORT_SEEK
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)